activejob/lib/active_job/enqueuing.rb

```diff
@@ -36,7 +36,7 @@ def job_or_instantiate(*args)
     #
     # ==== Examples
     #
-    #  class SiteScrapperJob < ActiveJob::Base
+    #  class SiteScraperJob < ActiveJob::Base
```

Comment:
We wanted it to Scrapper 😛 
 http://www.merriam-webster.com/dictionary/scrapper